src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1515,6 +1529,28 @@ private void OnCompileDotnetScript(MessageId msg_id, CancellationToken token)
             SendResponse(msg_id, Result.OkFromObject(new { }), token);
         }
 
+        private async Task<bool> OnContinueToLocation(MessageId sessionId, string script_id, int lineNumber, CancellationToken token)
+        {
+            DebugStore store = await RuntimeReady(sessionId, token);
+            ExecutionContext context = GetContext(sessionId);
+            Frame scope = context.CallStack.First<Frame>();
+            if (!SourceId.TryParse(script_id, out SourceId id))
+                return false;
+
+            var res = new List<SourceLocation>();
+
+            var sourceLocation = new SourceLocation(id, lineNumber, -1);
+
+            store.AddPossibleBreakpointsInMethodToList(sourceLocation, sourceLocation, res, scope.Method.Info);
```

Comment:
Should this use the script id to identify the method, instead of using `scope.Method`?

Reply:
And if it failed to add anything, then should this method fail?

Reply:
I don't think so, we can just move the cursor from the current method being executed. Also scriptId just tells us the file that is being executed and not the method.